### PR TITLE
Instructor Home page: show a message when a course has no sessions/evals, instead of having no panel body. #2346

### LINKED ISSUE #2346: Instructor Home page: show a message when a course has no sessions/evals, instead of having no panel body.

### PR DESCRIPTION
**Pull request by bamboo3250**
fixed #2346
@damithc This is what it looks like now. Is that okay?

instructorhome

This is what it was before #3917

### CODE REVIEW

**Comment by damithc** [comment]
The empty row doesn't look as tall as a usual non-empty row. Can we also show a message under the empty table?

**Comment by bamboo3250** [comment]
Other empty tables also look that way. So there will be a message under each empty table? There may be multiple courses in Instructor Homepage.

**Comment by damithc** [comment]
Yes, we can have a message in each empty course. The text can be smaller and in gray so that it doesn't attract too much attention. See if the the table looks better if the row size is adjusted to normal size. If it is better, then we should do it for all empty tables.

**Comment by bamboo3250** [comment]
@damithc which style do you prefer?

type1

type2

**Comment by damithc** [comment]
Can we have a combination?: i.e. have the empty row, but also have the message inside the panel.

**Comment by bamboo3250** [comment]
I guess you mean something like this?

instructorhome

**Comment by damithc** [comment]
I meant the message below the empty row (not inside the empty table), but this might work too. Anyway, can you do a screenshot of that too (i.e. message below the table)?

**Comment by junhaoyap** [comment]
Do you mean centralized below the table? There is already a screenshot of it being below the table but it is left aligned.

**Comment by damithc** [comment]
I meant the message is *inside* the panel, but below the table.

**Comment by bamboo3250** [comment]
Oh, I got what you mean. I'll upload the screenshot in a few minutes.

**Comment by bamboo3250** [comment]
@damithc How about this?

instructorhome2

**Comment by damithc** [comment]
The previous one (the one with the message inside in the table) looks neater. BTW, the message can be rephrased "This course does not have any sessions yet" because the course can have multiple instructors.

**Comment by bamboo3250** [comment]
So from now will we stick with this one?

instructorhome

**Comment by damithc** [comment]
Yes, that format but revised message text.

**Comment by bamboo3250** [comment]
Final screenshot

instructorhome3

@yj-soh could you review this PR as well? :sparkles:

File: src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag [hunk 1]
```diff
@@ -17,6 +17,18 @@
             <th class="no-print">Action(s)</th>
         </tr>
     </thead>
+    <c:if test="${empty sessionRows}">
+        <tr>
+            <td>
+                <span class="col-md-12 text-muted">
+                    This course does not have any sessions yet.
+                </span>
```
Comment (yj-soh) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: Is the `col-md-12` necessary?

Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: I fixed it

**Comment by yj-soh** [comment]
Looks fine.
Remember to test.

**Comment by bamboo3250** [comment]
thanks :+1:

**Pull request merged**